docs/IRISMapCubeSequence.rst

```diff
@@ -0,0 +1,187 @@
+
+This class inherits from NDCubeSequence_, so many methods of IRISMapCubeSequence are linked
+to their description in NDCubeSequence_ documentation.
+
+Initialization
+--------------
+
+To initialize an IRISMapCubeSequence, we will need to open some level 2 fits file.
+You can find these files in the IRIS_ website by selecting a file either by clicking
+on the map or by setting some information in the left panel. Now that we get some level 2
+fits file, we can load it into the reading method for fits files. This method will read
+all the fits file and will take all the information that it needs to create an
+IRISMapCubeSequence object.
+
+Let's assume that we will build an IRISMapCubeSequence with two fits files that we will
+call ``my_fits_file_0`` and ``my_fits_file_1``. We can create an IRISMapCubeSequence
+with as many fits files as we want but from the same observation. Next, we will call
+our IRISMapCubeSequence object as ``my_sequence``. ::
+
+    >>> from irispy import read_iris_sji_level2_fits
+    >>> my_fits_files = [my_fits_file_0, my_fits_file_1]
+    >>> my_sequence = read_iris_sji_level2_fits(my_fits_files)
+
+As for IRISMapCube, if you don't have a lot of RAM memory or if you are loading a huge file,
+we recommend to use the memmap kwarg. By using it, you will only load what you need to run
+but some methods that requires all the file will not be accessible. You can use the memmap
+kwarg by doing: ::
+
+    >>> from irispy import read_iris_sji_level2_fits
+    >>> my_fits_files = [my_fits_file_0, my_fits_file_1]
+    >>> my_sequence = read_iris_sji_level2_fits(my_fits_files, memmap=True)
+
+Class Structure
+---------------
+
+Sequence Attributes
+^^^^^^^^^^^^^^^^^^^
+
+So now, ``my_sequence`` is an IRISMapCubeSequence object and we can access to a lot of
+information like:
+
+- ``my_sequence.data`` : In this attribute, we can find the data array of our sequence.
+- ``my_sequence.meta`` : The meta data contains a lot of information which are not inside
+  the data array.
+- ``my_sequence.common_axis``: We can found some explanation in NDCubeSequence.Common_Axis_
+  paragraph.
+
+Cube Attributes
+^^^^^^^^^^^^^^^
+
+As an IRISMapCubeSequence object is a list of IRISMapCube objects, we can do the same things
+that we have seen previously in the IRISMapCube documentation. To access to the information
+stored in a IRISMapCube object, we can select the first IRISMapCube object by writting
+``my_sequence[0]`` or the Nth object with ``my_sequence[N-1]``. As they are IRISMapCube
+objects, we can access to their information by doing:
+
+- ``my_sequence[0].data`` : The data attribute of the first IRISMapCube object.
+- ``my_sequence[1].wcs`` : The WCS attribute of the second IRISMapCube object.
+- ``my_sequence[2].mask`` : The mask attribute of the third IRISMapCube object.
+- ``my_sequence[3].uncertainty`` : The uncertainty attribute of the fourth IRISMapCube object.
+
+Dimensions
+----------
+
```

Comment:
Since we overwrite the standard behaviour of ```NDCubeSequence.dimensions```, I think we should explicitly include a description and example of ```IRISMapCubeSequence.dimensions``` and ```IRISMapCubeSequence.world_axis_physical_types```

Reply:
Done. Effectively, this NDCube documentation don't describe our results.